### PR TITLE
fix(Pagination): add pagination props to toolbar

### PR DESCRIPTION
**Pull request by mtikhinskaya**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?**

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by acateland** [comment]
Please can you describe the purpose of this change what enhancement does it bring or what does it fix ?

**Comment by mtikhinskaya** [comment]
We use Pagination with it's props in Toolbar but doesn't list them in Toolbar.props

File: src/List/Toolbar/Pagination/Pagination.component.js [hunk 1]
```diff
@@ -116,7 +116,7 @@ Pagination.propTypes = {
 	id: PropTypes.string,
 	activePage: PropTypes.number,
 	itemsLength: PropTypes.number.isRequired,
-	onChangePagination: PropTypes.func.isRequired,
+	onChangePagination: PropTypes.func,
```
Comment (jmfrancois) [inline src/List/Toolbar/Pagination/Pagination.component.js]: If this function is not passed and the pagination is used I'm sure we will have an exception

Comment (mtikhinskaya) [inline src/List/Toolbar/Pagination/Pagination.component.js]: I just follow the pattern of `SelectDisplayMode.onSelectDisplayMode` and `SelectSortBy.onSelectSortBy`. We should have common pattern here

Comment (jmfrancois) [inline src/List/Toolbar/Pagination/Pagination.component.js]: True, https://github.com/Talend/react-talend-components/blob/45a6013521d4dd2fe8401cdc5d5164d5541c8cfc/src/List/Toolbar/SelectSortBy/SelectSortBy.component.js#L18

This is bad to have a widget without handler. but in the case of the SelectSortBy we may have the change on order without having the onSelectSortBy, which is a bit different.

So please add  *isRequired* back, I don't think the pagination may exists without the onChangePagination, this doesn't make sens

Comment (mtikhinskaya) [inline src/List/Toolbar/Pagination/Pagination.component.js]: This will be done in #171

**Comment by mtikhinskaya** [comment]
Will be fixed in #171

**Pull request closed without merging**